content/collections.md

```diff
@@ -0,0 +1,370 @@
+
+In Meteor, that data layer is typically stored in the Mongo database engine. A set of data in Mongo is referred to as a "collection", and in Meteor, it's [collections](http://docs.meteor.com/#/full/mongo_collection) that forms the persistence layer.
+
+However, collections are a lot more than a way to store data. They also provide the core of the realtime, connected user experience that the best applications provide, and which Meteor makes easy to implement.
+
+In this article, we'll look closely at how collections work in various places in the framework, and how to get the most out of them.
+
+### Server-side collections
+
+When you create a collection on the server:
+```js
+Todos = new Mongo.Collection('Todos');
+```
+
+You are creating a collection within Mongo itself, and a interface to that collection to be used on the server. It's a fairly straightforward layer on top of the underlying Node Mongo driver, but with a synchronous (fibers-based) API:
+
+```js
+// This line won't complete until the insert is done
+Todos.insert({_id: 'my-todo'});
+// So this line will return something
+const todo = Todos.findOne({_id: 'my-todo'});
+// Look ma, no callbacks!
+console.log(todo);
+```
+
+### Client-side collections
+
+On the client, when you write the same line:
+```js
+Todos = new Mongo.Collection('Todos');
+```
+
+It does something totally different!
+
+On the client, there is no direct connection to the Mongo database server, and in fact a sychronous API to it is not possible (nor probably what you want). Instead, on the client, a collection is a client side *cache* of the database, with a Mongo API. This is achieved thanks to the [Minimongo](https://www.meteor.com/mini-databases) library.
+
+Minimongo is an in-memory, all JS, Mongo implementation. What this means is that on the client, when you write:
+```js
+// This line is changing an in-memory datastructure
+Todos.insert({_id: 'my-todo'});
+// And this line is querying it
+const todo = Todos.findOne({_id: 'my-todo'});
+// So this happens right away!
+console.log(todo);
+```
+
+The way that you move data from the server (and Mongo-backed) collection into the client (in-memory) collection is the subject of the "data loading" article. But, generally speaking, you use a *subscription* to a *publication* to push data from the server to the client. Usually, you can assume that the client contains an up-to-date copy of some subset of the full Mongo collection.
+
+To write data back to the server, you use a *method*, the subject of the "methods and forms" article.
+
+### Local Collections
+
+There is a third way to use a collection in Meteor. On the client or server, if you create a collection but do *not* give it a name:
+
+```js
+Scratch = new Mongo.Collection(null);
```

Comment:
this code sample seems contrived - perhaps we can demo a simple situation where you might actually use this?